src/core/core.js

```diff
@@ -273,6 +273,9 @@ export default class Core {
         querySource: this.storage.get(StorageKeys.QUERY_SOURCE),
         additionalHttpHeaders: this._additionalHttpHeaders
       })
+      .catch(error => {
+        console.error('The following problem was encountered while executing a vertical search: ' + error);
```

Comment:
Would we want to have `SEARCH_COMPLETE` here as well? Otherwise, would the loading spinner just keep going?

Reply:
yes, I will add it there too! I just pushed some wip changes to deploy so I can test something with the new result update code in the site